addons/html_editor/static/src/fields/property_value.js

```diff
@@ -41,7 +41,7 @@ patch(PropertyValue.prototype, {
     },
 
     async onEditorBlur() {
-        const value = this.editor.getContent();
+        const value = markup(this.editor.getContent());
```

Comment:
This is another blind markup that is not showing why it would be safe here, it should be built into `getContent` instead.